python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -149,34 +139,46 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
                   .join(", "),
             },
             {
-              label: "Pending",
+              label: "Number of pending tasks",
               value: actor.taskQueueLength.toLocaleString(),
+              tooltip:
+                "The number of tasks that are currently pending to execute on this actor. If this number " +
+                "remains consistently high, it can indicate that this actor is a bottle-neck in your program.",
             },
             {
-              label: "Executed",
+              label: "Number of executed tasks",
               value: actor.numExecutedTasks.toLocaleString(),
+              tooltip:
+                "The number of tasks this actor has executed throughout its lifetimes.",
```

Comment:
Is plural "lifetimes" intentional? I think it resets when the actor is restarted.

Reply:
Oh. It was intentional. I was under the impression it did not reset when the actor is restarted. I'm not entirely sure where I got that idea from now though. I'll look into it.

Reply:
Confirmed that this does track across lifetimes.